### DISCUSSION TITLE
When using Pgpool and multiple replicas of Postgresql, create table using struct gets redirected to standby server.

### DISCUSSION THREAD

**Discussion by AtitShetty** [go-pg/pg#1937] [Q&A]
Hi,

I am using Pgpool with multiple replicas of Postgresql.

I am creating a table using struct. However, the query gets redirected to `standby` replica instead of `primary`.

I tried using `pgpool` configuration `primary_routing_query_pattern_list` and specified the table name too, but it did not work.

Has anyone faced the same issue?

I want to understand how the ORM converts the query before sending it to Postgresql, so that I can investigate further.

**Comment by mityaa** [comment]
Faced with it just now )
Have not you decided it yet?
